ninchatsdk/src/main/java/com/ninchat/sdk/views/NinchatWebRTCView.java

```diff
@@ -257,6 +261,24 @@ protected void startWithSDP(final JSONObject sdp) {
         }
     }
 
+    private void initializeAudioManager() {
+        // Create and audio manager that will take care of audio routing,
+        // audio modes, audio device enumeration etc.
+        ninAudioManager = NinAudioManager.create(videoContainer.getContext().getApplicationContext());
+        // Store existing audio settings and change audio mode to
+        // MODE_IN_COMMUNICATION for best possible VoIP performance.
+        Log.e(TAG, "Starting the audio manager...");
```

Comment:
This kind of logging should probably be verbose or debug rather than error

Reply:
Good catch. I was testing and forgot to remove that. 🤦‍♂  Thank you.